Extensions/PlatformBehavior/platformerobjectruntimebehavior.ts

```diff
@@ -548,14 +550,23 @@ namespace gdjs {
     }
 
     /**
-     * Mark the platformer object has not being grabbing any platform.
+     * Mark the platformer object as not grabbing any platform.
      */
     _releaseGrabbedPlatform() {
       if (this._state === this._grabbingPlatform) {
         this._setFalling();
       }
     }
 
+    /**
+     * Mark the platformer object as falling.
```

Comment:
Nitpicking: `Mark the platformer object as falling if on a ladder.`

(agree, the comment is now exactly the code, but it's a public interface so it's fine to comment it) (at least this means the code is crystal clear!)